docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,295 @@
+
+1. Check the statuses of the Mimir pods:
+
+   ```bash
+   kubectl -n mimir-test get pod
+   ```
+
+   The results look similar to this:
+
+   ```bash
+   kubectl  -n mimir-test get pod
+   NAME                                            READY   STATUS      RESTARTS   AGE
+   mimir-minio-78b59f5569-fhlhs                    1/1     Running     0          2m4s
+   mimir-nginx-74f8bff8dc-7kr7z                    1/1     Running     0          2m5s
+   mimir-mimir-distributed-make-bucket-job-z2hc8   0/1     Completed   0          2m4s
+   mimir-overrides-exporter-5fd94b745b-htrdr       1/1     Running     0          2m5s
+   mimir-query-frontend-68cbbfbfb5-pt2ng           1/1     Running     0          2m5s
+   mimir-ruler-56586c9774-28k7h                    1/1     Running     0          2m5s
+   mimir-querier-7894f6c5f9-pj9sp                  1/1     Running     0          2m5s
+   mimir-querier-7894f6c5f9-cwjf6                  1/1     Running     0          2m4s
+   mimir-alertmanager-0                            1/1     Running     0          2m4s
+   mimir-distributor-55745599b5-r26kr              1/1     Running     0          2m4s
+   mimir-compactor-0                               1/1     Running     0          2m4s
+   mimir-store-gateway-0                           1/1     Running     0          2m4s
+   mimir-ingester-1                                1/1     Running     0          2m4s
+   mimir-ingester-2                                1/1     Running     0          2m4s
+   mimir-ingester-0                                1/1     Running     0          2m4s
+   ```
+
+1. Wait until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
+
+## Configure Prometheus to write to Grafana Mimir
+
+Make a choice based on whether or not you already have a Prometheus server set up:
+
+- For an existing Prometheus server:
+
+  1. Add the following YAML snippet to your Prometheus configuration file:
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir, based on what is defined in the existing `scrape_configs` configuration.
+
+  1. Restart the Prometheus server.
+
+- For a Prometheus server that does not exist yet:
+
+  1. Write the following configuration to a `prometheus.yml` file:
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+
+     scrape_configs:
+       - job_name: prometheus
+         honor_labels: true
+         static_configs:
+           - targets: ["localhost:9090"]
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Start a Prometheus server by using Docker:
+
+     ```bash
+     docker run --network=host -p 9090:9090  -v <absolute-path-to>/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus
```

Comment:
Minor thing. Since we are using `--network=host` we can remove `-p 9090:9090` because it will be ignored anyway.